spec/util/apache_mod_platform_compatibility_spec.rb

```diff
@@ -12,10 +12,10 @@
   foobar_linux = 'foobar_linux'
 
   expected_compatible_platform_versions = {
-    'redhat' => [6, 7, 8],
-    'centos' => [6, 7, 8],
-    'oraclelinux' => [6, 7],
-    'scientific' => [6, 7],
+    'redhat' => [7, 8],
+    'centos' => [7, 8],
+    'oraclelinux' => [7],
+    'scientific' => [7],
     'debian' => [8, 9, 10],
```

Comment:
Is dropping 8 & 9 here scheduled for a future PR? As well as Ubuntu cleanup just below.

Reply:
We will be creating a separate thorough ticket just for removing OS support and their residual code wherever required. As for the part of this ticket, we have primarily focused on redhat family OS and that too Redhat , Scientific and OracleLinux 6.
This ticket is the successive ticket of https://github.com/puppetlabs/puppetlabs-apache/pull/2326/files
Anyway thanks for suggestion and reminding us.

Reply:
Makes sense, thanks.